plugins/cms-export/src/csv.ts

```diff
@@ -110,7 +110,7 @@ export function getDataForCSV(fields: CollectionField[], items: CollectionItem[]
                 }
 
                 default: {
-                    assertNever(field)
+                    shouldBeNever(field)
```

Comment:
Should we do `columns.push("")` here? It seems like we have still added this to the header.